alerts/openshift-container-storage-operator/CephOSDDown.md

```diff
@@ -0,0 +1,27 @@
+# CephOSDDown
+
+## Meaning
+
+OSD (Object Storage Daemon) are down indicates ceph-osd daemons not running as expected.
+
+## Impact
+
+**Severity:** Warning
+**Potential Customer Impact:** Medium
+
+This alert suggests that one or more OSDs might be down.
```

Comment:
The current impact section can be removed and replaced with what Travis mentioned in the [comment](https://github.com/openshift/runbooks/pull/233/files#r1946864105).  Something like: 

`Ceph detects that the OSDs are down and automatically start the recovery process by moving the data to other available OSDs. But if the OSDs having the copies of the data also fail during this backfilling, then there is a chance of permanent data loss. `

`Potential Customer Impact` can't be medium. Has to be High.  But I don't see other runbook files using `severity` and `Potential Customer Impact`. So not sure if can simply skip mentioning it.

Reply:
I have added impact as a warning because severity is also a warning.

Reply:
@rchikatw the main question is that other alerts don't have these levels under `impact` section. Why add these for this particular alert?